lib/_content.erb

```diff
@@ -0,0 +1,20 @@
+        <section class="doc-preview">
+                <%- @site_map['Documents'].each do |categories| -%>
+                        <%- categories.each do |key, value| -%>
+                                <%- first_entry = true -%>
+                                <%- value.each do |key2| -%>
+                                        <%- if key===@site_map.keys[0] && first_entry -%>
+                                                <%- active_keyword="active" -%>
+                                                <%- first_entry = false -%>
+                                        <%- else -%>
+                                                <%- active_keyword="" -%>
+                                        <%- end -%>
+
+                                        <div class="doc-preview__item <%=active_keyword%>" id="<%=key2["Slug"]%>">
```

Comment:
We would once again have to check for `reveal(raw(key2["Link"]))` to be true to prevent the divs of the documents that have broken links to appear on the html file.